tasks/config/run.js

```diff
@@ -177,6 +177,10 @@ module.exports = function () {
         'test/server_integration/http/ssl_redirect/config.js',
         '--config',
         'test/server_integration/http/cache/config.js',
+        '--config',
```

Comment:
@elastic/kibana-operations I don't see we run `serverIntegrationTests` task on CI. Or we have the task for other purposes?

Reply:
I don't see any indication that we're running this task on CI 😬

Reply:
I should it then, I suppose?

Reply:
I'll push up a PR to fix this separately if you like

Reply:
ok, I will merge it as is